--- docs/api/client-server/swagger-ui.js
+++ docs/api/client-server/swagger-ui.js
@@ -3150,7 +3150,7 @@
         html += '<div>' + addReference(schema, name) + '</div>';
       } else if (type === 'object') {
         if (_.isPlainObject(schema.properties)) {
-          html += _.map(schema.properties, function (property, name) {
+          var contents = _.map(schema.properties, function (property, name) {
             var propertyIsRequired = (_.indexOf(schema.required, name) >= 0);
             var cProperty = _.cloneDeep(property);
 
@@ -3201,7 +3201,9 @@
           }).join(',</div>');
         }
 
-        html += '</div>';
+        if (contents) {
+          html += contents + '</div>';
+        }
       } else {
         html += '<div>' + primitiveToOptionsHTML(schema, type) + '</div>';
       }
